src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,98 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final double DEFAULT_GROWTH_OF_ARRAY = 1.5;
+    private Object[] elements = new Object[DEFAULT_CAPACITY];
```

Comment:
```suggestion
    private T[] elements;
```
init instance fields (not constants) in constructor